cost/aws/reserved_instances/coverage/reserved_instance_coverage.pt

```diff
@@ -27,14 +37,22 @@ parameter "param_historical_number_of_days_in_past" do
   default "7"
 end
 
-auth "aws", type: "aws" do
-  version "4"
-  service "ce"
-  region "us-east-1"
-  access_key cred("AWS_ACCESS_KEY_ID")
-  secret_key cred("AWS_SECRET_ACCESS_KEY")
+###############################################################################
+# Authentication
+###############################################################################
+
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
```

Comment:
We need to also use the aws_sts scheme.  the schemes should be "aws","aws_sts"

Reply:
Updated the schemes part with "aws","aws_sts"